tools/cell_census_builder/__main__.py

```diff
@@ -5,15 +5,16 @@
 import os.path
 import sys
 from datetime import datetime, timezone
-from typing import List
+from typing import List, Union
 
 import pyarrow as pa
 import tiledbsoma as soma
+from anndata import AnnData
```

Comment:
unused.  why import?  also, how did this pass lint?

Reply:
I hadn't run pre-commit.  Finally did `pre-commit install`, so this shouldn't happen again.

Reply:
that explains why it was "OK" on your own machine.  But how did it pass CI when you pushed the commits?